src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +181,39 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssueSize() {
+        int count = 0;
```

Comment:
Should be `getClosedIssuesCount`? To be consistent of the rest of variable names.